inc/Config/ArraySerializable.php

```diff
@@ -51,5 +61,29 @@ public function to_array(): array {
 		return $this->config;
 	}
 
-	abstract protected static function get_config_schema(): array;
+	/**
+	 * @inheritDoc
+	 */
+	public static function preprocess_config( array $config ): array|WP_Error {
+		return $config;
```

Comment:
Is this a placeholder for migration code?

Reply:
Oh I see [from the lines below](https://github.com/Automattic/remote-data-blocks/blob/2e2b2af00d4a93d5b48b18ce4bc6a875e4927017/inc/Config/ArraySerializableInterface.php#L22-L30) this is meant to be used as an extendable method.

Reply:
Yes! But also the place to put / call migration code.